.eslintrc.js

```diff
@@ -75,7 +75,6 @@ module.exports = {
         ],
       },
     ],
-    'primer-react/no-deprecated-colors': ['warn', {checkAllStrings: true}],
```

Comment:
we removed this rule https://github.com/primer/eslint-plugin-primer-react/pull/174